flocker/control/_diffing.py

```diff
@@ -111,9 +427,19 @@ class Diff(PClass):
     changes = pvector_field(object)
 
     def apply(self, obj):
+        proxy = _TransformProxy(original=obj)
         for c in self.changes:
-            obj = c.apply(obj)
-        return obj
+            proxy = c.apply(proxy)
+        try:
+            return proxy.commit()
+        except:
```

Comment:
Is there a reason to use a bare except here? As opposed to e.g. Exception or BaseException.